awx/ui_next/src/components/AddDropDownButton/AddDropDownButton.jsx

```diff
@@ -30,25 +29,21 @@ function TemplateAddButton({ match, i18n }) {
         position={DropdownPosition.right}
         toggle={<ToolbarAddButton onClick={() => setIsOpen(!isOpen)} />}
         dropdownItems={[
-          <Link
-            key="job"
-            className="pf-c-dropdown__menu-item"
-            to={`${match.url}/job_template/add/`}
-          >
-            {i18n._(t`Job Template`)}
+          <Link key="top" className="pf-c-dropdown__menu-item" to={`${topUrl}`}>
+            {`${topLabel}`}
           </Link>,
           <Link
-            key="workflow"
+            key="bottom"
             className="pf-c-dropdown__menu-item"
-            to={`${match.url}_workflow/add/`}
+            to={`${bottomUrl}`}
           >
-            {i18n._(t`Workflow Template`)}
+            {`${bottomLabel}`}
           </Link>,
         ]}
       />
     </div>
   );
 }
 
-export { TemplateAddButton as _TemplateAddButton };
-export default withI18n()(withRouter(TemplateAddButton));
+export { AddDropDownButton as _AddDropDownButton };
+export default withI18n()(withRouter(AddDropDownButton));
```

Comment:
You should be able to remove `withI18n` and `withRouter` here since that has been moved up to the parent component